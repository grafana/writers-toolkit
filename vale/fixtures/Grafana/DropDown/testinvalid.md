Don't use:

- dropdown
- drop down
