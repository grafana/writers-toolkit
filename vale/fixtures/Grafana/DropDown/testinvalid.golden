testinvalid.md:3:3:Grafana.DropDown:Use _drop-down_ rather than _dropdown_ or _drop down_. Use drop-down as a modifier rather than as a standalone noun. For example: _drop-down menu_. 
testinvalid.md:4:3:Grafana.DropDown:Use _drop-down_ rather than _dropdown_ or _drop down_. Use drop-down as a modifier rather than as a standalone noun. For example: _drop-down menu_. 
